vnext/codegen/react/components/rnwcore/EventEmitters.cpp

```diff
@@ -0,0 +1,149 @@
+
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @generated by codegen project: GenerateEventEmitterCpp.js
+ */
+
+#include <react/renderer/components/rnwcore/EventEmitters.h>
+
+namespace facebook {
+namespace react {
+
+void ModalHostViewEventEmitter::onRequestClose(OnRequestClose event) const {
```

Comment:
could all these be noexcept?

Reply:
This code is generated by core, so not something thats going to change in this PR. And calls into a lot of non-noexcept code.  I'm going to go with no for now. -- But its something we can look into later.